src/media/css/base.styl

```diff
@@ -154,5 +154,42 @@ body {
 }
 
 #site-footer {
```

Comment:
enforcing a no ID standard will prevent specificity issues. even though for this one element we may not foresee needing to override anything, it's just easier to use classes for everything